src/common/utxobased/engine/makeServerStates.ts

```diff
@@ -337,10 +337,6 @@ export function makeServerStates(config: ServerStateConfig): ServerStates {
           .then(response => {
             if (!resolved) {
               resolved = true
-              if ('error' in response) {
```

Comment:
This commit should have been broken up in two. One for the renames and one that actually changes code